### PR TITLE
Release surveys and READ ME clarification.

### PR DESCRIPTION
**Pull request by ryyeung**
Added releases survey to Pulsar and Advanced CLI. Made note in README to include version-docs and docs folders to confuse less users.

### CODE REVIEW

File: docs/protocol/substrate-cli.mdx [hunk 1]
```diff
@@ -23,6 +23,10 @@ import { AdvancedSystemdServiceFileGenerator } from '@site/src/components/System
 [Pulsar](https://github.com/subspace/pulsar) (previously Simple CLI or Subspace CLI) is the recommended way to farm on the Subspace Network. Follow the guide below to get started or check out the project [README on GitHub](https://github.com/subspace/pulsar/blob/main/README.md).
 :::
 
+:::tip Feedback!
```
Comment (jim-counter) [inline docs/protocol/substrate-cli.mdx:26]:
```suggestion
:::note Feedback!
```

File: versioned_docs/version-latest/protocol/substrate-cli.mdx [hunk 2]
```diff
@@ -23,6 +23,10 @@ import { AdvancedSystemdServiceFileGenerator } from '@site/src/components/System
 [Pulsar](https://github.com/subspace/pulsar) (previously Simple CLI or Subspace CLI) is the recommended way to farm on the Subspace Network. Follow the guide below to get started or check out the project [README on GitHub](https://github.com/subspace/pulsar/blob/main/README.md).
 :::
 
+:::tip Feedback!
```
Comment (jim-counter) [inline versioned_docs/version-latest/protocol/substrate-cli.mdx:26]:
```suggestion
:::note Feedback!
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Makes sense to standardise on either `tip` or `note`.

**Review by EmilFattakhov — COMMENTED** [review]
Agree, that we should be more persistent with the type of banner we use. 

That you for adding the information about versioning to readme, Ryan!

**Pull request merged**